packages/notifi-axios-adapter/lib/queries/getDiscordTargetsImpl.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  collectDependencies,
+  makeParameterLessRequest,
+} from '@notifi-network/notifi-axios-utils';
+import { GetDiscordTargetsResult } from '@notifi-network/notifi-core';
+
+import {
+  discordTargetFragment,
+  discordTargetFragmentDependencies,
+} from '../fragments';
+
+const DEPENDENCIES = [
+  ...discordTargetFragmentDependencies,
+  discordTargetFragment,
+];
+
+const MUTATION = `
```

Comment:
Not sure if it should be called `QUERY` ?

Reply:
you're right. that is confusing. let me do a sweep and check